docs/learn/dapp-developer/mint-lsp7-token.md

```diff
@@ -1,6 +1,7 @@
 ---
 sidebar_label: '💽 Mint LSP7 Token'
 sidebar_position: 8
+description: Learn how to mint LSP7 tokens
```

Comment:
You have 155 characters to put as many keywords as possible. Lets make use of them.

Add LUKSO, Digital Asset etc.


```suggestion
description: Learn how to create Digital Assets (token) on LUKSO using the LSP7 standard.
```